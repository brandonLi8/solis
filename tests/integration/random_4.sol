# Copyright Â© 2022-2023 Brandon Li. All rights reserved.

##
Random test of a Solis program that uses ints, bools, and all the operators that are associated with them.
NOTE: this file was auto-generated with the `ints_and_bools_random.py` script.
##

-(((-((-(39.67843541479574)) + (-((let a1: int = -((let b1: int = 64) / (b1))) / (-(420)))))) + (b1)) / (-(let c1: float = let d1: float = (-(579)) * (-((-(a1)) + (-(8.172797181135445)))))));
-(d1);
(-(c1)) / (-(36.826507849324265));
-(d1);
-(let e1: float = c1);
-(e1);
(let f1: float = 22.01651053806731) * (-(f1));
e1;
-((e1) * (-(d1)));
-(((let g1: float = -(e1)) * ((-(782)) + (-(let h1: float = e1)))) + (-((-(let i1: float = -(d1))) - (-((-(let j1: int = 94)) - (-(let k1: float = (e1) * (i1))))))));
-(30.097430791495796);
k1;
-(let l1: float = -(h1));
g1;
f1;
-(g1);
g1;
c1;
-(let m1: float = 35.898910685335494);
-(let n1: float = -(m1));
0.9062342172596404;
-(n1);
-(let o1: float = (g1) + ((let p1: int = j1) % (-(j1))));
k1;
-(m1);
k1;
-(c1);
d1;
-(h1);
let q1: float = e1;
l1;
q1;
d1;
m1;
d1;
((-(let r1: float = 19.048464124892206)) / (-(d1))) + ((-(let s1: int = -((b1) + (b1)))) % (263));
h1;
-((-(let t1: float = f1)) + (-(let u1: float = -(h1))));
-(let v1: float = -(n1));
u1;
-(l1);
m1;
-((-(2.973162753661377)) + (-((-(j1)) % (-(42)))));
h1;
18.50776136008728;
-((376) * (e1));
(-(l1)) - (-((-(a1)) % (b1)));
l1;
r1;
g1;
-(t1);
l1;
-(i1);
t1;
-(18.323666513434134);
-(f1);
i1;
let w1: float = k1;
r1;
-(15.536674494879131);
25.260784169446893;
-(q1);
d1;
-(o1);
d1;
i1;
-(r1);
(a1) / (let x1: float = -(let a2: float = -(w1)));
-(g1);
-(let b2: float = o1);
l1;
-(i1);
c1;
m1;
-(l1);
-(((-(i1)) * (5.250320061633625)) + ((285) * (423)));
r1;
let c2: float = -(k1);
let d2: float = 29.702559440215037;
0.46386523132842616;
18.674655447609023;
-(d2);
-(w1);
b2;
-(43.974157495193374);
-(let e2: float = i1);
(-(5.167043147901091)) - (-(a1));
(48.13369025972528) + (-(d2));
-(v1);
-(i1);
-(r1);
-(16.431688112021693);
12.056212444845746;
-(19.626356035970282);
(-((-(35.80779602060495)) / (-(u1)))) * ((-(340)) - (let f2: float = f1));
-(c2);
q1;
-(q1);
-(let g2: float = w1);
o1;
-(t1);
-((-(n1)) + (b1));
20.400598634242993;
-(b2);
-(21.08921685388663);
-(3.131919079610122);
-(let h2: float = -(b2));
a2;
-(5.4283504911740375);
-(k1);
l1;
26.166833754129687;
(d2) - (-(12.34270909717967));
-(l1);
d2;
-(let i2: float = -(e2));
a2;
e2;
let j2: float = -(k1);
h2;
r1;
n1;
-(15.201424343503591);
(h1) * (-((-(p1)) / (s1)));
let k2: float = -(49.3660384562262);
-(19.56048321338879);
-(h1);
n1;
13.381595390793686;
-(1.7760087798662114);
let l2: float = -(r1);
-((-(275)) - (-(i1)));
l1;
-(t1);
(534) + (e1);
-(q1);
l2;
(-(880)) * (i1);
let m2: float = c1;
let n2: float = -(o1);
d2;
-((let o2: float = -(o1)) - (-(let p2: float = -(let q2: float = let r2: float = -(let s2: float = -((-(f1)) * ((586) * (p1))))))));
r1;
-(let t2: float = -(v1));
let u2: float = 28.856794686780873;
let v2: float = -(36.189276854828925);
-(r2);
w1;
48.81119914725279;
-(q1);
v2;
-(s2);
-(r2);
25.73600610755147;
((a1) / (-(d1))) + (-(d1));
-((n2) * (-(a1)));
f2;
-((-(let w2: int = -((-(s1)) * (-(a1))))) + (-(t1)));
-(i2);
a2;
r1;
k1;
-(let x2: float = -(e2));
-(k2);
-(i1);
let a3: float = -(i2);
r2;
-((-(26.307481872253252)) - (v2));
-(u2);
let b3: float = -(f1);
a2;
let c3: float = g1;
-(let d3: float = e2);
(d2) / (-(let e3: int = -((let f3: int = -(j1)) + (let g3: int = -(a1)))));
-((let h3: float = d2) + (-(35.16146215533235)));
let i3: float = 2.33861772809716;
-(13.402864235573624);
-(t2);
-(12.604250394837868);
(-((-((-(b1)) - (g3))) + (-(43.84747201117155)))) / (-(v1));
-(m1);
-(let j3: float = -(v2));
-(u1);
-(g1);
i2;
18.638764780472975;
-((-(r2)) - (w1));
1.091879897573612;
-(let k3: float = let l3: float = -(let m3: float = -(let n3: float = 14.271524614151321)));
-(n3);
k1;
-(e2);
a2;
6.9081789956613235;
-((-(d2)) / (-((let o3: float = -(t1)) * (-(let p3: float = j3)))));
i1;
t2;
-(let q3: float = let r3: float = -(u2));
i2;
-((f2) + (let s3: int = let t3: int = -(let u3: int = -(((819) * (-(((-((345) * (-(let v3: int = -(927))))) - ((g3) * (s1))) - (let w3: int = (let x3: int = j1) - (-(a1)))))) / (-(b1))))));
b1 + a1 + d1 + c1 + e1 + f1 + g1 + h1 + i1 + j1 + k1 + l1 + m1 + n1 + p1 + o1 + q1 + r1 + s1 + t1 + u1 + v1 + w1 + a2 + x1 + b2 + c2 + d2 + e2 + f2 + g2 + h2 + i2 + j2 + k2 + l2 + m2 + n2 + o2 + s2 + r2 + q2 + p2 + t2 + u2 + v2 + w2 + x2 + a3 + b3 + c3 + d3 + f3 + g3 + e3 + h3 + i3 + j3 + n3 + m3 + l3 + k3 + o3 + p3 + r3 + q3 + v3 + x3 + w3 + u3 + t3 + s3