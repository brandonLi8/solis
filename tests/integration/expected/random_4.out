-4506877116781714884